vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -99,6 +99,7 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClInclude Include="ReactPackageProvider.h" />
+    <ClInclude Include="AutolinkedNativeModules.g.h" />
```

Comment:
The other PR has this for C# too, just checking if it is not needed to backport C# too?

Reply:
It's already there, it's only in the cpp file that it was missing (since it still works to have includes that aren't actually mentioned)